### PR TITLE
Preserving the table structure.

### PR DESCRIPTION
**Pull request by joe247**
Thank you for accepting the pull request.

The code <> syntax in README.md preserves the table structure. I'll edit it later and try to bring in the original markdown table format.

### CODE REVIEW

**Comment by joe247** [comment]
This does not look good
This does not look good

For the time being... this looks better
For the time being... this looks better

>The code <> syntax in README.md preserves the table structure. I'll edit it later and try to bring in the original markdown table format.

**Comment by mike-fabian** [comment]
This pull request changes  a lot  more stuff in README.md than just putting the tables into the markdown for code.

I think I’ll take only the markdown for the code.

File: README.md [hunk 1]
```diff
@@ -1,94 +1,92 @@
-# Ibus-typing-booster
+ibus-typing-booster
```
Comment (mike-fabian) [inline README.md]: https://guides.github.com/features/mastering-markdown/
```
Headers

# This is an <h1> tag
## This is an <h2> tag
###### This is an <h6> tag
```
So why are you removing the #?

File: README.md [hunk 2]
```diff
@@ -1,94 +1,92 @@
...
+  input methods).<br>
+- Several input methods and languages can be used at the same time without switching.<br>
+- Predicts Unicode symbols and emoji as well.
+
+Requirements
+------------
+- Some packages are requied if you are building from source:
+```
+    ┏━━━━━━━━━━━━━━━━━━━┯━━━━━━━━━┓
+    ┃      Package      │ Version ┃
```
Comment (mike-fabian) [inline README.md]: Your build  requirements  table is only for Ubuntu, it is quite different for Fedora,  openSUSE, ...

http://mike-fabian.github.io/ibus-typing-booster/development.html

tries to list the build requirements for different distributions, I would rather update the Ubuntu section there if
you think something is missing there.

**Comment by mike-fabian** [comment]
I included the quoting  of the tables as code in https://github.com/mike-fabian/ibus-typing-booster/releases/tag/2.6.7  but not your other changes.

File: README.md (hunk 1 above)
Comment (joe247) [reply README.md]: Oh, oops! my mistake.

File: README.md (hunk 2 above)
Comment (joe247) [reply README.md]: Ok, I see.

**Pull request closed without merging**